packages/deployer/config/secrets.env.example

```diff
@@ -5,3 +5,6 @@ SS_CURRENCY_CONVERTER_API_CURRENCY_CONVERTER_SECURITY_SCHEME=""
 HEARTBEAT_API_KEY=""
 HEARTBEAT_ID=""
 HEARTBEAT_URL=""
+
+CHAINAPI_TESTING_API_KEY=""
```

Comment:
Forgot to add, I don't think Airnode should be hardcoding any references to ChainAPI. Can this be done with a single environment variable?

Reply:
I also think that just "TESTING_" by itself doesn't necessarily accurately describe the key as "testing" can mean any number of things.

Maybe `TEST_ENDPOINT_API_KEY`, `ENDPOINT_TESTING_API_KEY`  or something similar instead? Probably want to get some input from the rest of the team

Reply:
Making two env variables was @bbenligiray 's idea. One can be used by ChainAPI and one by the user deploying the airnode for som custom access if need be. Not sure if it's a use case worth supporting though. Maybe we can stick with just one for now and see if somebody actually needs multiple API keys.